app/src/components/blocks/lloydGeorgeRecordError/LloydGeorgeRecordError.tsx

```diff
@@ -48,21 +50,27 @@ function LloydGeorgeRecordError({ downloadStage, setStage }: Props) {
         return (
             <span>
                 <h3 data-testid="no-records-title">No records available for this patient </h3>
-                <p>
-                    You can upload full or part of a patient record. You can upload supporting files
-                    once the record is uploaded.
-                </p>
-                <div className="lloydgeorge_record-stage_header-content-no_record">
-                    <ButtonLink
-                        className="lloydgeorge_record-stage_header-content-no_record-upload"
-                        data-testid="upload-patient-record-button"
-                        onClick={() => {
-                            navigate(routes.LLOYD_GEORGE_UPLOAD);
-                        }}
-                    >
-                        Upload patient record
-                    </ButtonLink>
-                </div>
+                {featureFlags.uploadLloydGeorgeWorkflowEnabled &&
```

Comment:
Can you move this to line 46-48 with the other conditions?

Reply:
I think the h3 on line 52 needs to be displayed regardless of the feature flags - if I move the conditions I've added up, then it isn't displayed